src/Admin/AbstractAdminExtension.php

```diff
@@ -79,6 +79,9 @@ public function getPersistentParameters(AdminInterface $admin)
         return [];
     }
 
+    /**
+     * @return array<string, string|string[]>
```

Comment:
Isn't this better on the Interface ?

Reply:
This is not possible, yet. There is only a `@method` definition for this.

Reply:
Oh yes indeed, didn't see my code was on master ^^' 

Should we add the phpdoc to the comment on the Interface then ?
```
    /**
     * Return the controller access mapping.
     */
    // NEXT_MAJOR: Uncomment this method
    // public function getAccessMapping(AdminInterface $admin): array;
```